client/src/components/AppNavbar.js

```diff
@@ -43,11 +43,19 @@ class AppNavBar extends Component {
                     Gallery
                   </NavLink>
                 </NavItem>
+
+                <NavItem className="">
+                  <NavLink tag={Link} to="/Associations">
+                    Associations
+                  </NavLink>
+                </NavItem>
+
                 <NavItem className="">
                   <NavLink tag={Link} to="/lifecoaching">
                     Life Coaching
                   </NavLink>
                 </NavItem>
+
                 {!this.props.loggedIn ? (
                   <>
                     <NavItem>
```

Comment:
Don't leave spaces between the Nav Items.

Reply:
Fixed